### DISCUSSION TITLE
Learning

### DISCUSSION THREAD

**Discussion by peachkalemarsexpress** [monkeytypegame/monkeytype#3762] [Ideas]
It would be very cool to add the ability to translate a word after you type it.
This would give you the opportunity to learn the language as well.)

**Comment by Jeff0502** [comment]
Maybe this can be done by creating specific text files. For example, a file called Afrikaans - English, in which you first type the Afrikaans translation and then the English translation, without having to add any new code. Like:

"boek - book" 

Or

"Hierdie is a sin - This is a sentence"

**Reply by Raphael2b3** [reply]
aswell you could use an open translation api that prefetches the words and dynamicly to easly support any language.

**Reply by Raphael2b3** [reply]
for japanese romanji we can use this website and webscrape the translation from the response:
[https://jisho.org/search/naomi](https://jisho.org/search/naomi)

because i want the feature for japanese i ll focus on that at first.
